src/com/сompanyzayats/modul/UserInputService.java

```diff
@@ -0,0 +1,47 @@
+package modul;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class UserInputService {
+    private final Scanner scanner = new Scanner(System.in);
+    private final Actions actions = new Actions();
+
+    public int[] arrayFill() {
```

Comment:
If the method does several actions, this should be displayed in the name. createAndFillArray